presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -114,6 +114,7 @@
     private int fileSystemMaxCacheSize = 1000;
 
     private boolean writesToNonManagedTablesEnabled;
+    private boolean tableStatisticsEnabled;
```

Comment:
I guess this is the reason why there are product test failures related to table statistics.